core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -538,6 +538,25 @@ interface Debug {
 
     @ConfigGroup
     interface Compression {
+        /**
+         * Whether compression should be enabled.
+         */
+        @WithDefault("true")
+        boolean enabled();
+
+        /**
+         * Whether the compression should be executed within a container.
+         */
+        Optional<Boolean> containerBuild();
```

Comment:
```suggestion
        boolean containerBuild();
```
why is this one optional, having default an empty optional which interpreted as `false`, or a boolean default init with `false` would turn out the same, right? And make `.orElse(false)` obsolete. 

Or is the optional used not to enforce a declaration on the user side? 

Im not familiar with this. sorry for questioning.